client/css/bootstrap.css

```diff
@@ -8,197 +8,6 @@
  * Generated using the Bootstrap Customizer (http://getbootstrap.com/customize/?id=242b77ebf84da0cd12cf)
  * Config saved to config.json and https://gist.github.com/242b77ebf84da0cd12cf
  */
-/*! normalize.css v3.0.1 | MIT License | git.io/normalize */
```

Comment:
Per my comment below, I highly suggest we revert this until getting rid of Bootstrap as a whole, if that ever happens.